tests/conftest.py

```diff
@@ -17,9 +17,21 @@
 
 """File conftest.py for pytest test suite."""
 
+import random
+
 import pytest
+import flexmock
+
+from thoth.adviser.context import Context
+from thoth.adviser.beam import Beam
+from thoth.adviser.enums import RecommendationType
+from thoth.adviser.pipeline_config import PipelineConfig
+from thoth.adviser.predictor import Predictor
+from thoth.adviser.resolver import Resolver
+from thoth.adviser.predictor import Predictor
```

Comment:
`from thoth.adviser.predictor import Predictor` twice

Reply:
Thanks! Addressed in 61c83eb